### ISSUE TITLE
cloudflare_zone_lockdown documentation's example doesn't work in terraform 0.12

### ISSUE THREAD

**Issue by jakshi** [terraform-providers/terraform-provider-cloudflare#406]
### Terraform Version

```
$ terraform --version
Terraform v0.12.3
+ provider.cloudflare v1.16.0
```

### Affected Resource(s)

cloudflare_zone_lockdown documentation

### Expected Behavior
Example from https://www.terraform.io/docs/providers/cloudflare/r/zone_lockdown.html wouldn't produce syntax error on terraform 0.12.

### Actual Behavior
If you try to apply example from documentation you get error about configuration block absence.

### Steps to Reproduce

Put example to .tf file
Configure cloudflare provider
run `terraform init`
run `terraform plan`

### Example from documentation

```
resource "cloudflare_zone_lockdown" "endpoint_lockdown" {
  zone        = "api.mysite.com"
  paused      = "false"
  description = "Restrict access to these endpoints to requests from a known IP address"
  urls = [
    "api.mysite.com/some/endpoint*",
  ]
  configurations = [
    {
      "target" = "ip"
      "value" = "198.51.100.4"
    },
  ]
}
```

### Working example

```
resource "cloudflare_zone_lockdown" "endpoint_lockdown" {
  zone        = "api.mysite.com"
  paused      = "false"
  description = "Restrict access to these endpoints to requests from a known IP address"
  urls = [
    "api.mysite.com/some/endpoint*",
  ]
  configurations {
      target = "ip"
      value = "198.51.100.4"
    }
}
```

**Comment by jacobbednarz** [comment]
PRs are welcome if you’d like to tackle this one.

**Comment by jakshi** [comment]
https://github.com/terraform-providers/terraform-provider-cloudflare/pull/407/files

**Comment by patryk** [comment]
Fixed in #407
